nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -741,9 +741,8 @@ in {
     { assertions = [
       { assertion = cfg.database.createLocally -> cfg.config.dbpassFile == null;
         message = ''
-          Using `services.nextcloud.database.createLocally` (that now defaults
-          to true) with database password authentication is no longer
-          supported.
+          Using `services.nextcloud.database.createLocally` with database
+          password authentication is no longer supported.
```

Comment:
Even though it's already mentioned in the docs, I'd probably add a note here that you're supposed to use socket auth now.

Reply:
Actually, this is covered by the paragraph right below:
```
          If you want this module to manage your nextcloud database for you,
          unset `services.nextcloud.config.dbpassFile` and
          `services.nextcloud.config.dbhost` to use socket authentication
          instead of password.
``` 

 Should I duplicate that info anyway?

Reply:
Nvm, my bad!